Dockerfile

```diff
@@ -48,9 +52,8 @@ RUN cd /kb/dev_container/modules && \
     sed -i 's/print .*keeping.*/#ignore/'  /kb/deployment/lib/GenomeTypeObject.pm
 
 
-#RUN sed -i 's/capture_stderr/tee_stderr/' /kb/deployment/lib/Bio/KBase/GenomeAnnotation/GenomeAnnotationImpl.pm
-
 RUN \
+    cpanm Array::Utils && \
     cpanm install Set::IntervalTree && \
```

Comment:
Put all the perl installation stuff together unless there is a good reason not to.